managed/resource_cluster.go

```diff
@@ -308,6 +303,7 @@ func createClusterSpec(ctx context.Context, plan Cluster, clusterExists bool) (c
 		if clusterType == "SYNCHRONOUS" {
 			info.PlacementInfo.SetMultiZone(false)
 		}
+		info.SetIsDefault(false)
```

Comment:
What is this for?

Reply:
It shows the error `IsDefault` has to be fault for multi-region clusters, I am not exactly sure of the reason.

Reply:
The default value was changed from `true` to `false` here https://github.com/yugabyte/yugabyte-cloud/commit/3c49c65e19d36b444f745b2b8b6035348cdc0b46#diff-4ea2fce92e955dee25f775c4ad6467aac4d54cf4ca03dfdf09a27509ed58082cL3654-R3657. That is why, we will have to make this change.